lib/taskana-core/src/test/java/acceptance/task/ServiceLevelPriorityAccTest.java

```diff
@@ -297,22 +297,25 @@ void testSetPlannedPropertyOnSingleTaskWithTaskUpdate()
       userName = "admin",
       groupNames = {"group_2"})
   @Test
-  void testSetDuePropertyOnSingleTask()
+  void should_SetDueOrPlannedProperty_when_TaskUpdateIsCalled()
```

Comment:
Would it be too much effort to separate the cases into separate tests? I would prefer that.

Reply:
Done

Reply:
Done